eva-accession-import/src/main/java/uk/ac/ebi/eva/accession/dbsnp/model/Orientation.java

```diff
@@ -34,4 +40,8 @@ public static Orientation getOrientation(Integer orientation) {
             throw new IllegalArgumentException("Orientation must be +1, -1 or null");
```

Comment:
Consider re-phrasing to "Orientation must be +1, -1 or unknown"

Reply:
👍